src/Resources/contao/classes/BackendUser.php

```diff
@@ -154,39 +158,6 @@ public function __get($strKey)
 		return parent::__get($strKey);
 	}
 
-
-	/**
-	 * Redirect to the login screen if authentication fails
-	 *
-	 * @return boolean True if the user could be authenticated
-	 */
-	public function authenticate()
```

Comment:
The redirect is missing in the parent class... in fact, the redirect is the only thing which made `BackendUser::authenticate()` almost unusable for the outside world, however this is an BC break if this method now will behave different.
We do not have to use it in the core anymore but it has to remain for existing extensions which call it and expect the redirect and script termination to happen.